internal/proxy/impl.go

```diff
@@ -2657,7 +2669,13 @@ func (node *Proxy) Flush(ctx context.Context, request *milvuspb.FlushRequest) (*
 // Query get the records by primary keys.
 func (node *Proxy) Query(ctx context.Context, request *milvuspb.QueryRequest) (*milvuspb.QueryResults, error) {
 	receiveSize := proto.Size(request)
-	metrics.ProxyReceiveBytes.WithLabelValues(strconv.FormatInt(paramtable.GetNodeID(), 10), metrics.QueryLabel).Add(float64(receiveSize))
+	metrics.ProxyReceiveBytes.WithLabelValues(
+		strconv.FormatInt(paramtable.GetNodeID(), 10),
+		metrics.QueryLabel, request.GetCollectionName()).Add(float64(receiveSize))
+
+	metrics.ProxyReceivedNQ.WithLabelValues(
```

Comment:
Why ProxyReceivedNQ metrics be added when performing a query request.

Reply:
We consider a query to be a request with nq=1.